src/pages/Address/AddressAssetComp.tsx

```diff
@@ -74,16 +86,25 @@ export const AddressSporeComp = ({ account }: { account: Spore }) => {
   const id = formatNftDisplayId(amount, 'spore')
   return (
     <AddressAssetComp
+      isRGBPP={isRGBPP}
       href={`/nft-collections/${collection?.typeHash}`}
       property={`id: ${id.slice(0, 8)}...${id.slice(-8)}`}
       name={sliceNftName(symbol)}
-      udtLabel="Spore"
+      udtLabel={udtLabel ?? 'Spore'}
       icon={{ url: patchMibaoImg(icon), errorHandler: handleNftImgError }}
     />
   )
 }
 
-export const AddressMNFTComp = ({ account }: { account: MNFT }) => {
+export const AddressMNFTComp = ({
+  account,
+  isRGBPP,
+  udtLabel,
```

Comment:
It's `m nft comp` so the label should be 'm nft'

Reply:
done